.circleci/config.yml

```diff
@@ -1,58 +1,39 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
   build:
     docker:
-      # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
       - image: python:3-stretch
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
     working_directory: ~/robot
 
     steps:
       - checkout
 
-      # Download and cache dependencies
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "requirements.txt" }}
-          - v1-dependencies-{{ checksum "robotd/requirements.txt" }}
-          # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-
       - run:
           name: install dependencies
           command: |
             python3 -m venv venv
             . venv/bin/activate
       - run: apt-get update -y
       - run: apt-get install -y libcv-dev libopencv-photo-dev libopencv-contrib-dev libusb-1.0.0-dev
+      - run: pip3 install git+https://github.com/sourcebots/sb-vision@8ace07108ee27bf86e3a59603f40dd62f6e65101
+      - run: pip3 install git+https://github.com/sourcebots/robotd@9992ff6d169365ab807df1965cbc1662e350ca04
       - run: pip3 install -r requirements.txt
       - run: pip3 install -r script/linting/requirements.txt
       - run: pip3 install -r script/typing/requirements.txt
-      - run: git clone git@github.com:sourcebots/sb-vision.git
-      - run: cd sb-vision; pip3 install .
-      - run: git clone git@github.com:sourcebots/robotd.git
-      - run: cd robotd; pip3 install .
       - save_cache:
           paths:
             - ./venv
           key: v1-dependencies-{{ checksum "requirements.txt" }}
 
-      # run tests!
       - run:
           name: run tests
           command: python3 -m unittest
-
+          
```

Comment:
[nitpick]: trailing whitespace

Reply:
oh god, how the hell did that get there?! 😱